src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -329,6 +333,15 @@
                     ]
                 }
             ]
+        },
+        {
+            "Id": "EnhancedHealthReportingEnabled",
+            "Name": "Enable Enhanced Health Reporting",
+            "Description": "Enhanced health reporting provides free real-time application and operating system monitoring of the instances and other resources in your environment. Do you want to enable enhanced health reporting?",
+            "Type": "Bool",
```

Comment:
Should we model this as a string with allowed values? That way if Beanstalk ever adds another possible value for `aws:elasticbeanstalk:healthreporting:system,SystemType` we wouldn't have to make a breaking change to support.

Reply:
Done